common/data/src/main/java/org/thingsboard/server/common/data/notification/targets/platform/UsersFilterType.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright © 2016-2023 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.thingsboard.server.common.data.notification.targets.platform;
+
+public enum UsersFilterType {
+    USER_LIST,
+    CUSTOMER_USERS,
+    ALL_USERS,
+    ORIGINATOR_ENTITY_OWNER_USERS
+
+//    USER_GROUP,
```

Comment:
remove commented block

Reply:
Please add TENANT_ADMINISTRATORS (all users with tenant administrator authority for current tenant id). 

We also need to discuss creating notifications on behalf of the SYS_ADMIN.